src/web/app/pages-student/student-profile-page/student-profile-page.component.html

```diff
@@ -8,7 +8,8 @@ <h1>{{ name }}</h1>
           <img
               src="{{ profilePicLink }}"
               class="profile-pic"
-              (click)="onUploadEdit()">
+              (click)="onUploadEdit()"
+              (error)="this.profilePicLink = this.DEFAULT_PICTURE_LINK">
```

Comment:
`DEFAULT_PICTURE_LINK` is not defined?

Reply:
Oh yah, updated! I will thoroughly check all my changes and make sure everything is working now :P